pom.xml

```diff
@@ -1861,6 +1861,17 @@
                             <fileset>
                                 <directory>${project.basedir}/web-ui/dist</directory>
                             </fileset>
+                            <fileset>
+                                <!-- Clean up the extension spark jars, to avoid include unspecified version -->
+                                <directory>${maven.multiModuleProjectDirectory}/extensions/spark</directory>
+                                <includes>
+                                    <include>kyuubi-extension-spark-*/target/kyuubi-extension-spark-*.jar</include>
+                                </includes>
+                                <excludes>
+                                    <exclude>kyuubi-extension-spark-common/target/*.jar</exclude>
+                                    <exclude>kyuubi-extension-spark-jdbc-dialect/target/*.jar</exclude>
```

Comment:
this is not necessary, as `kyuubi-extension-spark-jdbc-dialect` is always activated.